dockstore-webservice/src/main/java/io/dockstore/webservice/languages/ServicePrototypePlugin.java

```diff
@@ -63,6 +64,19 @@ public VersionTypeValidation validateWorkflowSet(String initialPath, String cont
                 if (files == null) {
                     validationMessageObject.put(initialPath, "The key 'files' does not exist.");
                     isValid = false;
+                } else {
+                    // check that files in .dockstore.yml exist
+                    ArrayList<String> missingFiles = new ArrayList<String>();
```

Comment:
If so inclined, you could rewrite this and the for loop with map and filter. But it's fine the way it is.

Feel free to ignore this; I'm just personally liking the FP style more.